packages/fxa-settings/src/components/Security/index.test.tsx

```diff
@@ -27,8 +27,8 @@ describe('Security', () => {
       </AppContext.Provider>
     );
 
-    expect(await screen.findByText('rk-header')).toBeTruthy;
-    expect(await screen.findByText('tfa-row-header')).toBeTruthy;
+    expect(await screen.findByText('Recovery key')).toBeTruthy;
+    expect(await screen.findByText('Two-step authentication')).toBeTruthy;
```

Comment:
Ref: https://github.com/mozilla/fxa/issues/12868

Curiously, I don't see this file in the results from https://github.com/mozilla/fxa/issues/12868#issuecomment-1125528607 so I'm forced to assume my grep was a giant dumpster fire.

I do find this result if I do the grosser <kbd>git grep -n ".toBe" packages | grep -Ev ");?$" | grep ";"</kbd> but I'm even less confident in that solution. I might have to play a bit more and see if the Jest ESLint rules catch these.